packages/presentation_layer/lib/app/screens/palindrome/palindrome_bloc/palindrome_bloc.dart

```diff
@@ -10,11 +10,13 @@ abstract class PalindromeBloc extends Bloc {
   void updateState();
   void updateCurrentInputValue(String value);
   void verifyString();
+  void updateLoadingStatus({required bool loading});
 }
 
 class _PalindromeBlocImpl extends BlocImpl implements PalindromeBloc {
   final PalindromeUsecaseImpl _palindromeUsecase;
   final PalindromeData _state = PalindromeData.init();
+  bool _isLoading = false;
```

Comment:
as a variant - move isLoading into abstraction BlocImpl. in this case u'll have no need in defining variable everywhere

Reply:
corrected